italics-quotes-a-to-f: Apply correct use of italics and quotes to a-z glossary

dfitzmau opened the pull request:
Issue: https://github.com/redhat-documentation/supplementary-style-guide/issues/173

Related Jira(s):

- https://github.com/redhat-documentation/supplementary-style-guide/pull/215

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:
```diff
@@ -61,7 +61,7 @@
 [[active-directory-forest]]
 ==== image:images/yes.png[yes] Active Directory forest (noun)
-*Description*: An Active Directory (AD) forest is a set of one or more domain trees which share a common global catalog, directory schema, logical structure, and directory configuration. The forest represents the security boundary within which users, computers, groups, and other objects are accessible.
+*Description*: An _Active Directory_ (AD) forest is a set of one or more domain trees which share a common global catalog, directory schema, logical structure, and directory configuration. The forest represents the security boundary within which users, computers, groups, and other objects are accessible.
```
I think you'd need to include "forest" in the italics here

dfitzmau replied:
Updated.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc:
```diff
@@ -85,7 +85,7 @@
 [[active-directory-security-identifier]]
 ==== image:images/yes.png[yes] Active Directory security identifier (noun)
-*Description*: A security identifier (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
+*Description*: A _security identifier_ (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
```
Might be out of scope of this PR, but wondering if this should say

```suggestion
*Description*: An _Active Directory security identifier_ (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
```

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc:
```diff
@@ -13,7 +13,7 @@
 [[backtrace]]
 ==== image:images/yes.png[yes] backtrace (noun)
-*Description*: "Backtrace" is another name for "stack trace" (or "stack backtrace"), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term "traceback", possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. "Traceback" is the ...
+*Description*: _Backtrace_ is another name for _stack trace_ (or _stack backtrace_), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term _traceback_, possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. _Traceback_ is the ...
```
I have to say that I'm uncertain whether the two "traceback" instances in the last two sentences should be italics or quotes. I'm leaning toward quotes because we're talking about what term to use, but could use some other thoughts here.

dfitzmau replied:
Yes, good point. Both sentences do suggest on how to use the term. I updated with quotes.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc:
```diff
@@ -127,9 +127,9 @@
+*Description*: _Bimodal IT_ is the Gartner phrase for the combination of traditional (mode 1 or type 1) and modern (mode 2 or type 2) IT infrastructure and resources. There are many ways to talk about this combination approach. Using only the Gartner term can alienate other analysts or those not familiar with Gartner's phrasing.
 
-The practice of having both modes together is often referred to as "hybrid", "agile", or "modern" IT. "Hybrid IT" is a more general term; for example, it could mean "on-premise plus public cloud". "Agile" and "modern IT" can both carry an implication of "mode 2". When using those terms, be specific about the exact technology combination you mean.
+The practice of having both modes together is often referred to as _hybrid_, _agile_, or _modern_ IT. _Hybrid IT_ is a more general term; for example, it could mean _on-premise plus public cloud_. _Agile_ and _modern IT_ can both carry an implication of _mode 2_. When using those terms, be specific about the exact technology combination you mean.
```
This is another one that I'm thinking that quotes here was actually correct. But I'm not positive

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -141,7 +141,7 @@
 [[ceph-block-device]]
 ==== image:images/yes.png[yes] Ceph Block Device (noun)
-*Description*: In Red Hat Ceph Storage, the Ceph Block Device is the block storage component of Ceph. Also known as the RADOS Block Device, however the term Ceph Block Device is preferred.
+*Description*: In Red Hat Ceph Storage, the _Ceph Block Device_ is the block storage component of Ceph. Also known as the _RADOS Block Device_, however the term _Ceph Block Device_ is preferred.
```
Another thought that the last instance of "Ceph Block Device" should be in quotes

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -177,7 +177,7 @@
 [[ceph-object-gateway]]
 ==== image:images/yes.png[yes] Ceph Object Gateway (noun)
-*Description*: In Red Hat Ceph Storage, the Ceph Object Gateway is the S3/Swift component. Also known as RADOS gateway. However, prefer using the Ceph Object Gateway.
+*Description*: In Red Hat Ceph Storage, the _Ceph Object Gateway_ is the S3/Swift component. Also known as _R_ADOS gateway_. However, prefer using the _Ceph Object Gateway_.
```
The underscore is in the wrong spot here for `_R_ADOS gateway_`. Also the last instance of "Ceph Object Gateway" should probably be in quotes

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -444,7 +444,7 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 [[cloudwashing]]
 ==== image:images/yes.png[yes] cloudwashing (verb)
-*Description*: "Cloudwashing" is the process of rebranding legacy products to include the term "cloud" to increase their appeal to the cloud market.
+*Description*: _Cloudwashing_ is the process of rebranding legacy products to include the term _cloud_ to increase their appeal to the cloud market.
```
I'd leave "cloud" in quotes here

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -680,7 +680,7 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 [[connector]]
 ==== image:images/yes.png[yes] connector (noun)
-*Description*: 1) In Red Hat AMQ, a connector is a configurable entity for AMQ brokers and routers. They define an outgoing connection from either a router to another endpoint, or from a broker to another endpoint. 2) In Fuse Ignite, a connector provides a template for creating any number of connections to a particular application or service, each of which can perform a different operation. A Cam ...
+*Description*: 1) In Red Hat AMQ, a _connector_ is a configurable entity for AMQ brokers and routers. They define an outgoing connection from either a router to another endpoint, or from a broker to another endpoint. 2) In Fuse Ignite, a connector provides a template for creating any number of connections to a particular application or service, each of which can perform a different operation. A _ ...
```
Is "Camel component" really a term that needs to be italicized here?

dfitzmau replied:
Removed italics and lowercased "camel" :heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -707,7 +707,7 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 [[consumer]]
 ==== image:images/yes.png[yes] consumer (noun)
-*Description*: 1) In an LDAP replication environment, consumers receive data from suppliers or hubs. 2) In Red Hat AMQ, a consumer is a client that receives messages. 3) In Red Hat Fuse, a consumer is an endpoint that acts as the source of message exchanges entering a route. It wraps received messages in an exchange and then sends the exchange to the next node in the route. A route can have only  ...
+*Description*: 1) In an LDAP replication environment, _consumers_ receive data from suppliers or hubs. 2) In Red Hat AMQ, a consumer is a client that receives messages. 3) In Red Hat Fuse, a _consumer_ is an endpoint that acts as the source of message exchanges entering a route. It wraps received messages in an exchange and then sends the exchange to the next node in the route. A route can have o ...
```
Could you italicize the 2nd definition of "consumer" too?

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -756,7 +756,7 @@ image repository contains one or more tagged images.
 [[container-based]]
 ==== image:images/yes.png[yes] container-based (adjective)
-*Description*: Use "container-based" as an adjective when referring to applications made up of multiple services that are distributed in containers. "Container-based" can be used interchangeably with "containerized".
+*Description*: Use _container-based_ as an adjective when referring to applications made up of multiple services that are distributed in containers. "Container-based" can be used interchangeably with "containerized".
```
This should stay in quotes since it's giving direction on how to use the term, not defining it.

dfitzmau replied:
:white_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -129,9 +129,9 @@
 [[data-warehouse]]
 ==== image:images/yes.png[yes] Data Warehouse (noun)
-*Description*: In Red Hat Virtualization, the Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
+*Description*: In Red Hat Virtualization, the _Manager_ includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
```
I don't think Manager needs to be italicized here since it's not the term we're introducing

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -331,7 +331,7 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 [[devops-n]]
 ==== image:images/yes.png[yes] DevOps (noun)
-*Description*: "DevOps" is a combination of "Development" and "Operations". It refers to a specific method or organizational approach where developers and IT operations staff work together to create the applications that run the business.
+*Description*: _DevOps_ is a combination of Development and Operations. It refers to a specific method or organizational approach where developers and IT operations staff work together to create the applications that run the business.
```
I think the quotes around "Development" and "Operations" were probably appropriate

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -613,7 +613,7 @@ Red Hat Directory Server conforms to LDAP standards.
 [[drools-expert]]
 ==== image:images/yes.png[yes] Drools Expert (noun)
-*Description*: The "Drools Expert" is a pattern matching-based rule engine that runs on Java EE application servers, Red Hat JBoss BRMS platform, or bundled with Java applications. It comprises an inference engine, a production memory, and a working memory. Rules are stored in the production memory, and the facts that the inference engine matches the rules against are stored in the working memory ...
+*Description*: The +Drools Expert+ is a pattern matching-based rule engine that runs on Java EE application servers, Red Hat JBoss BRMS platform, or bundled with Java applications. It comprises an inference engine, a production memory, and a working memory. Rules are stored in the production memory, and the facts that the inference engine matches the rules against are stored in the working memory ...
```
The plus signs here should be underscores

dfitzmau replied:
:+1:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc:
```diff
@@ -2,7 +2,7 @@
 [[ee]]
 ==== image:images/yes.png[yes] ee subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "ee" subsystem is used to configure functionality in the Jakarta Enterprise Edition platform. In general text, write in lowercase as one word. Use "EE subsystem" when referring to the ee subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _ee subsystem_ is used to configure functionality in the Jakarta Enterprise Edition platform. In general text, write in lowercase as one word. Use "EE subsystem" when referring to the ee subsystem in titles and headings.
```
In other entries, on the EAP subsystem name was italicized. I'm thinking those probably should include "subsystem" in italics like this one does. (Ex: batch-jberet) if you wouldn't mind sweeping through those in the a-f files?

dfitzmau replied:
Fixed from entries a-f. :+1:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc:
```diff
@@ -427,7 +427,7 @@ The practice of having both modes together is often referred to as "hybrid", "ag
 [[build-configuration]]
 ==== image:images/yes.png[yes] build config (noun)
-*Description*: In Red Hat OpenShift, a build config describes a single build definition and a set of triggers for when a new build should be created. The API object for a build config is `BuildConfig`.
+*Description*: In Red Hat OpenShift, a _build config_ describes a single build definition and a set of triggers for when a new build should be created. The API object for a build config is _BuildConfig_.
```
Would you mind putting the last `BuildConfig` in backticks. (consistent with other openshift terms that do the same)

dfitzmau replied:
:white_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc:
```diff
@@ -236,7 +236,7 @@ Do not use "Firewire" or "firewire". Although FireWire is a trademark of Apple C
 [[futexes]]
 ==== image:images/yes.png[yes] futexes (noun)
-*Description*: "Futex" is an abbreviation of "fast user-space mutex". "Futexes" is the correct plural form.
+*Description*: _Futex_ is an abbreviation of _fast user-space mutex_. _Futexes_ is the correct plural form.
```
Can you make the two first sentences of "fuxed" and "futexes" match? They are slightly different (with one using italics and one using quotes for what it stands for).

dfitzmau replied:
:heavy_check_mark:

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc:
```diff
@@ -247,7 +247,7 @@ Do not use "Firewire" or "firewire". Although FireWire is a trademark of Apple C
 [[fuzzy]]
 ==== image:images/caution.png[with caution] fuzzy (adjective)
-*Description*: It is only correct to use "fuzzy" as an adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and exa ...
+*Description*: It is only correct to use _fuzzy_ as an adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and exa ...
```
This should stay quotes since it's telling how it should be written

dfitzmau replied:
:heavy_check_mark:

bergerhoffer reviewed (commented):
@dfitzmau Thanks for working on this! Added some feedback.

The only other thing I'm uncertain about is whether all of the terms that have abbreviations in parentheses afterward should be included in italics or not. 

@redhat-documentation/ccs-style-council Does anyone know?

Which should it be:
* an _access control instruction_ (ACI) grants
* an _access control instruction (ACI)_ grants

dfitzmau commented:
> @dfitzmau Thanks for working on this! Added some feedback.
> 
> The only other thing I'm uncertain about is whether all of the terms that have abbreviations in parentheses afterward should be included in italics or not.
> 
> @redhat-documentation/ccs-style-council Does anyone know?
> 
> Which should it be:
> 
> * an _access control instruction_ (ACI) grants
> * an _access control instruction (ACI)_ grants

Thanks, @bergerhoffer .

I implemented your feedback. Just back from PTO, so sorry for the delay.

I added italics around abbreviations. Please let me know your thoughts.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc:
```diff
@@ -691,7 +691,7 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 [[connector]]
 ==== image:images/yes.png[yes] connector (noun)
-*Description*: 1) In Red Hat AMQ, a connector is a configurable entity for AMQ brokers and routers. They define an outgoing connection from either a router to another endpoint, or from a broker to another endpoint. 2) In Fuse Ignite, a connector provides a template for creating any number of connections to a particular application or service, each of which can perform a different operation. A Cam ...
+*Description*: 1) In Red Hat AMQ, a _connector_ is a configurable entity for AMQ brokers and routers. They define an outgoing connection from either a router to another endpoint, or from a broker to another endpoint. 2) In Fuse Ignite, a connector provides a template for creating any number of connections to a particular application or service, each of which can perform a different operation. A c ...
```
I think Camel was correct capitalized as it was

dfitzmau replied:
Reverted _Camel_ to its original state.

bergerhoffer commented on supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc:
```diff
@@ -129,9 +129,9 @@
 [[data-warehouse]]
 ==== image:images/yes.png[yes] Data Warehouse (noun)
-*Description*: In Red Hat Virtualization, the Manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
+*Description*: In Red Hat Virtualization, the manager includes a comprehensive management history database, which any application can use to extract a range of information at the data center, cluster, and host levels. Installing Data Warehouse creates the `ovirt_engine_history` database, to which the Manager is configured to log information for reporting purposes.
```
I think Manager was correct capitalized as it was (it's capitalized in other RHV entries).

dfitzmau replied:
OK, I am confused by this one, as _manager_ is usually a generic noun. I feel a more specific name is missing in this section, such as Data Warehouse Manager or Red Hat Virtualization Manager. 


Outside the scope of this PR, so I reverted _Manager_ to its original state.

bergerhoffer replied:
Yeah I think in the context of RHV here, "Manager" is capitalized. See the guidance here: https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-virtualization-manager

bergerhoffer reviewed (commented):
Just two capitalization things, then this will LGTM.

This is a hard update because some of them aren't clear-cut whether it should be italics or quotes, but I think this is definitely an improvement!

dfitzmau commented:
> Just two capitalization things, then this will LGTM.
> 
> This is a hard update because some of them aren't clear-cut whether it should be italics or quotes, but I think this is definitely an improvement!

Thanks, @bergerhoffer , for the reviews. I implemented your feedback. Please let me know your thoughts.

bergerhoffer reviewed (dismissed):
Thanks @dfitzmau this LGTM now!

@redhat-documentation/ccs-style-council ready for more reviews please!

dfitzmau commented:
> Thanks @dfitzmau this LGTM now!
> 
> @redhat-documentation/ccs-style-council ready for more reviews please!

Thanks. I updated subsystem entries to ensure italics were used correctly to highlight these terms.

bergerhoffer commented:
@dfitzmau My only question left is do we need the backticks that you added for later in the descriptions? I didn't do that part to my PR, so let me know if you think it's needed and I'll add it to mine, or otherwise it should be reverted from this one. wdyt?

bergerhoffer reviewed (dismissed):
You answered my question in the other PR, so this LGTM. But it looks like there are conflicts now, likely due to the BIOS entry being updated, if you can resolve.

dfitzmau commented:
> @dfitzmau My only question left is do we need the backticks that you added for later in the descriptions? I didn't do that part to my PR, so let me know if you think it's needed and I'll add it to mine, or otherwise it should be reverted from this one. wdyt?

Yes, this would be great. Subsystem names in a general context refer to a specific component of a product. I usually adopt backticks for such instances on any products I work on. Thanks.

dfitzmau commented:
> You answered my question in the other PR, so this LGTM. But it looks like there are conflicts now, likely due to the BIOS entry being updated, if you can resolve.

Thanks, @bergerhoffer . I always am delighted when I merge conflict goes well! I updated the PR.

bergerhoffer reviewed (approved):
Approving!

@redhat-documentation/ccs-style-council Ready for more reviews!

inoxx03 reviewed (approved):
I left 3 small comments after my review. The comments are not directly relevant to the goal of this PR, so I do not want to hold off merging the PR because of them. I propose that we take note of them and fix them separately in a follow-up PR

bburt-rh commented:
This has three approvals, so I'm merging per @dfitzmau's request in gchat. He will open new PRs for the additional items brought up by @inoxx03
